enterprise_gateway/services/processproxies/docker_swarm.py

```diff
@@ -89,14 +89,14 @@ def get_container_status(self, iteration: int | None) -> str:
         """Return current container state."""
         # Locates the kernel container using the kernel_id filter.  If the status indicates an initial state we
         # should be able to get at the NetworksAttachments and determine the associated container's IP address.
-        task_state = None
+        task_state = ""
```

Comment:
This file is missing an implementation for 
```python
def get_error_states(self):
```

Reply:
they are not in use here, do you still want me to add them here?